package.json

```diff
@@ -29,5 +29,8 @@
     "@types/node": "^14.0.25",
     "protractor": "^7.0.0",
     "typescript": "^3.9.7"
+  },
+  "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2"
```

Comment:
install this dependency as dev dependency (`npm install --save-dev`)

Reply:
all dependencies must be installed as dev ?

Reply:
yes